apps/storefront/src/utils/b3Product/b3Product.ts

```diff
@@ -943,18 +967,41 @@ export const getVariantInfoDisplayPrice = (
   const variant = newVariants ? newVariants.find((item: Variant) => item.sku === variantSku) : {};
 
   if (variant && variant?.sku) {
-    const { purchasing_disabled: purchasingDisabled, inventory_level: inventoryLevel } = variant;
+    const {
+      purchasing_disabled: purchasingDisabled,
+      inventory_level: inventoryLevel,
+      available_to_sell: availableToSell,
+      unlimited_backorder: unlimitedBackorder,
+    } = variant;
```

Comment:
Could you prefix the name of these variables with `variant` like `variantAvailableToSell`